auto_round/autoround.py

```diff
@@ -140,11 +141,12 @@ def __init__(
             dynamic_max_gap: int = -1,
             data_type: str = "int",
             scale_dtype: str = "fp16",
-            multimodal:bool = False,
             act_bits: int = 32,
             act_group_size: int = None,
             act_sym: bool = None,
             act_dynamic: bool = True,
+            multimodal:bool = False,
+            quant_vision:bool = False,
```

Comment:
how about just pass the block_names [list of list] to support this, while the example of mutlimodel could support this args